wskadmin-next configuration error

The tool `wskadmin-next` failed to list namespaces when following the steps in the [README](https://github.com/apache/incubator-openwhisk/blob/master/tools/admin/README-NEXT.md).

## Environment details:

* local deployment with docker-compose
* version
  * docker:18.03.1-ce
  * docker-compose:1.22.0
  * Ubuntu: 18.04 (4.14.47-64)
  
## Steps to reproduce the issue:

1.   Local deployment with docker-compose
2.   [Build](https://github.com/apache/incubator-openwhisk/blob/master/tools/admin/README-NEXT.md#build) wskadmin-next
3.   Add [configuration file](https://github.com/apache/incubator-openwhisk/blob/master/tools/admin/README-NEXT.md#build)
4.   Run `$./wskadmin-next -c application-cli.conf user get guest`


## Expected results:

```
guest  23bc46b1-71f6-4ed5-8c54-816aa4f8c502:123zO3xZCLrMN6v2BKK1dXYFpXlPkccOFqm12CdAsMgRU4VrNZ9lyGVCGuMDGIwP
```


## Actual results:

```
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:51)
Caused by: java.lang.ExceptionInInitializerError
	at whisk.common.TransactionId$.started$extension(TransactionId.scala:89)
	at whisk.core.database.CouchDbRestStore.get(CouchDbRestStore.scala:221)
	at whisk.core.database.UserCommand.getKey(UserCommand.scala:223)
	at whisk.core.database.UserCommand.exec(UserCommand.scala:158)
	at whisk.core.cli.WhiskAdmin.executeCommand(Main.scala:190)
	at whisk.core.cli.Main$$anonfun$5.apply(Main.scala:107)
	at whisk.core.cli.Main$$anonfun$5.apply(Main.scala:106)
	at scala.util.Try$.apply(Try.scala:192)
	at whisk.core.cli.Main$.executeWithSystem(Main.scala:106)
	at whisk.core.cli.Main$.execute(Main.scala:93)
	at whisk.core.cli.Main$.main(Main.scala:86)
	at whisk.core.cli.Main.main(Main.scala)
	... 8 more
Caused by: pureconfig.error.ConfigReaderException: Cannot convert configuration to a whisk.common.tracing.TracingConfig. Failures are:
  at 'whisk.tracing':
    - (jar:file$OPENWHISK_PROJECT_HOME/bin/wskadmin-next!/BOOT-INF/lib/openwhisk-common-1.0.0-SNAPSHOT.jar!/application.conf:193) Key not found: 'component'.

	at pureconfig.package$.getResultOrThrow(package.scala:138)
	at pureconfig.package$.loadConfigOrThrow(package.scala:160)
	at whisk.common.tracing.WhiskTracerProvider$.<init>(OpenTracingProvider.scala:158)
	at whisk.common.tracing.WhiskTracerProvider$.<clinit>(OpenTracingProvider.scala:158)
	... 20 more

```

--- a/tools/admin/src/main/resources/application.conf
+++ b/tools/admin/src/main/resources/application.conf
@@ -1,0 +1,9 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
+# license agreements; and to You under the Apache License, Version 2.0.
+
+whisk{
+  # tracing configuration
+  tracing {
+    component = "cli"
+  }
+}

